composer.json

```diff
@@ -16,10 +16,10 @@
         }
     },
     "require": {
-        "drupal/bootstrap": "^3.23",
+        "drupal/gin": "^3.0",
         "drupal/core": "^8",
         "os2forms/os2forms": "^2.6",
-        "os2forms/os2forms_forloeb": "^1.3",
+        "os2forms/os2forms_forloeb": "dev-remove_rules_modules_from_forloeb",
```

Comment:
I don't think a specific branch of the module should be here. It should only be a version constraint similar to the one used before.

Reply:
Making this change straight away as the blocking PR is now merged.